sdk/dotnet/Pulumi/Rpc/ResourceField.cs

```diff
@@ -0,0 +1,19 @@
+﻿// Copyright 2016-2018, Pulumi Corporation
+
+#nullable enable
+
+using System;
+
+namespace Pulumi.Rpc
+{
+    [AttributeUsage(AttributeTargets.Field)]
+    public class ResourceFieldAttribute : Attribute 

```

Comment:
https://docs.microsoft.com/en-us/dotnet/standard/design-guidelines/attributes

> **✓ DO** seal custom attribute classes, if possible. This makes the look-up for the attribute faster.

```suggestion
    public sealed class ResourceFieldAttribute : Attribute 
```

Reply:
Done.